moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -137,13 +138,18 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     pose_reference_frame_ = getRobotModel()->getModelFrame();
 
     trajectory_event_publisher_ = pnode_->create_publisher<std_msgs::msg::String>(
-        trajectory_execution_manager::TrajectoryExecutionManager::EXECUTION_EVENT_TOPIC, 1);
+        rclcpp::names::append(opt_.move_group_namespace_,
```

Comment:
Could we document what this does?
```suggestion
        // Append the slash between two topic components
        rclcpp::names::append(opt_.move_group_namespace_,
```